app/gui/view/documentation/assets/styles.css

```diff
@@ -0,0 +1,218 @@
+/* Custom CSS rules used by the documentation panel. */
+
+/* Common parts. */
+
+:root {
+  --type-name-color: #9640da;
+  --module-name-color: #a239e2;
+  --methods-header-color: #1f71d3;
+  --method-name-color: #1f71d3;
+  --types-header-color: #1f71d3;
+  --examples-header-color: #6da85e;
+  --important-background-color: #edefe7;
+  --info-background-color: #e6f1f8;
+  --example-background-color: #e6f1f8;
+  --docs-background-color: #fcfeff;
+  --docs-text-color: #434343;
+  --tag-background-color: #f5f5f5;
+  --caption-background-color: #0077f6;
+}
+
+.enso-docs {
+  font-family: "M PLUS 1", DejaVuSansMonoBook, sans-serif;
+  font-size: 11.5px;
+  color: var(--docs-text-color);
+  background-color: var(--docs-background-color);
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+/* It's better to use `.unordered-list` class here instead of `ul`, but it will break the keyed lists that a provided by the engine.
+ * We need a separate way to represent keyed lists without embedded HTML tags. */
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  list-style-position: inside;
+}
+
+ul li:before {
+  content: "•";
+  font-size: 13px;
+  font-weight: 700;
+  margin-right: 3px;
+}
+
+ul li.type-item:before {
+  color: var(--type-name-color);
+}
+
+ul li.method-item:before {
+  color: var(--method-name-color);
+}
+
+.paragraph {
+  margin: 0;
+}
+
+.link {
+  cursor: pointer;
+}
+
+.link:hover {
+  text-decoration: underline;
+}
+
+.method {
+  color: var(--method-name-color);
+  font-weight: 600;
+}
+
+.constructor {
+  color: var(--type-name-color);
+  font-weight: 600;
+}
+
+.type {
+  color: var(--type-name-color);
+  font-weight: 600;
+}
+
+.entry-name {
+  opacity: 0.85;
+}
+
+.arguments {
+  opacity: 0.34;
+}
+
+.section-content {
+  padding-left: 1.25rem;
+}
+
+/* Headers. */
+
+.header-top {
+  font-size: 17px;
+  font-weight: 700;
+  color: var(--type-name-color);
+  margin: 0.5rem 0;
+}
+
+.header-icon {
+  pointer-events: none;
+  width: 0.85em;
+  height: 0.85em;
+  margin: 0.1em 0.05em 0 0.05em;
+  fill: none;
+  align-self: flex-start;
+  padding-top: 0.25em;
+}
+
+.header-container {
+  display: flex;
```

Comment:
1. Are you using any divs that are not using `flex/grid` layout? If not, all divs should use `grid` (or `flex` if `grid` is not possible).
2. Grid layout is faster than flex, if you can use it, please do.

Reply:
> all divs should use grid (or flex if grid is not possible)

What is the reason behind this? The purpose of several divs in the current layout is to provide necessary styles (by attaching CSS classes), and for grouping stuff. `grid` layout doesn't fit the documentation panel, as most of it content is rather "1D" (either vertically or horizontally), and `flex` is used only in necessary places.

Reply:
The reason is performance. Flexbox is faster for layout as it has very simplified rules of layouting.

Reply:
Ok, I checked, and I think the amount of regular divs is considerably low. Adding flex rules everywhere will complicate future support, so I'm abstaining from it. Let's return to this if performance will be concerning.